example/assets/templates/street-view-window.html

```diff
@@ -0,0 +1,4 @@
+<div ng-controller="WinController">
+    <ui-gmap-street-view-panorama focalCoord="parameter" radius="50" >
```

Comment:
example just happens to use ui-gmap-window and marker, but it does not require the map at all.